agent/agent.go

```diff
@@ -75,7 +75,7 @@ func (a *orbAgent) startBackends() error {
 			return errors.New("specified backend does not exist: " + name)
 		}
 		be := backend.GetBackend(name)
-		config["scrape_otel"] = strconv.FormatBool(a.config.OrbAgent.Otel.Scrape)
+		config["scrape_otel"] = strconv.FormatBool(a.config.OrbAgent.Otel.Enable)
```

Comment:
I didn't notice the placement of this before - this is not very clean. I would rather add another parameter to Backend.Configure() interface and pass in the whole Otel config (after the backend config, like `Configure(*zap.Logger, policies.PolicyRepo, map[string]string, map[string]string) error` or move this config from `orb.otel.enable` to `orb.backends.pktvisor.otel.enable` so that it's in the existing config structure